spec/models/instance/settings/effective_settings_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+RSpec.describe Instance::Settings::EffectiveSettings, type: :model do
+  let(:settings) do
+    mock_settings(components: {
+                    A: { enabled: true }
+                  })
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.